var/spack/repos/builtin/packages/omnitrace/package.py

```diff
@@ -12,12 +12,31 @@ class Omnitrace(CMakePackage):
     """Application Profiling, Tracing, and Analysis"""
 
     homepage = "https://amdresearch.github.io/omnitrace"
-    git = "https://github.com/AMDResearch/omnitrace.git"
-    maintainers("jrmadsen")
+    git = "https://github.com/ROCm/omnitrace.git"
+    url = "https://github.com/ROCm/omnitrace/archive/refs/tags/rocm-6.2.0.tar.gz"
+    maintainers("dgaliffiAMD", "afzpatel", "srekolam", "renjithravindrankannath", "jrmadsen")
 
     license("MIT")
 
+    version("amd-mainline", branch="amd-mainline", submodules=True)
+    version("amd-staging", branch="amd-staging", submodules=True)
     version("main", branch="main", submodules=True)
```

Comment:
I don't think that you  need to include the "main" branch anymore. The default branch is now "amd-staging" and the older commit hashes will also be in this branch.

Reply:
Ok, I removed it, I also deprecated the older versions of omnitrace